From e17c41ab9c34ea8715c2655fcb623b0bb92eaab1 Mon Sep 17 00:00:00 2001
From:  R, Dharageswari <dharageswari.r@intel.com>
Date: Thu, 29 Apr 2010 20:30:16 +0530
Subject: [PATCH] ADR-Post-Beta-0.05.002.03-8/8-Moorestown Audio Drivers: Config files

This patch adds the makefiles and Kconfig changes for both SST and MAD drivers

Signed-off-by: Vinod Koul <vinod.koul@intel.com>

	modified:   sound/pci/Kconfig
	modified:   sound/pci/Makefile
	new file:   sound/pci/sst/Makefile
Patch-mainline: 2.6.35?
---
 sound/pci/Kconfig      |   25 +++++++++++++++++++++++++
 sound/pci/Makefile     |    3 ++-
 sound/pci/sst/Makefile |    9 +++++++++
 3 files changed, 36 insertions(+), 1 deletions(-)
 create mode 100644 sound/pci/sst/Makefile

diff --git a/sound/pci/Kconfig b/sound/pci/Kconfig
index 1513d72..e41380d 100644
--- a/sound/pci/Kconfig
+++ b/sound/pci/Kconfig
@@ -8,6 +8,30 @@ menuconfig SND_PCI
 	  Support for sound devices connected via the PCI bus.
 
 if SND_PCI
+config SND_INTEL_SST
+	bool "Moorestown SST (LPE) Driver"
+	depends on X86 && LNW_IPC
+	default y
+	help
+	  Say Y here to include support for the Moorestown SST DSP driver
+	  On other PC platforms if you are unsure answer 'N'
+
+config SND_INTELMID
+	bool "Moorestown sound card driver"
+	select SND_JACK
+	depends on SND_INTEL_SST && SPI_MRST
+	default y
+	help
+	  Say Y here to include support for the Moorestown sound driver.
+	  On other PC platforms if you are unsure answer 'N'
+ 
+config SND_AUDIO_DBG_PRINT
+	bool "Moorestown Audio driver debug printk calls"
+	depends on SND_INTELMID
+	default n
+	help
+	   Say Y here to include debug printks calls in the Intel MID driver.
+	   If you are unsure say 'N'
 
 config SND_AD1889
 	tristate "Analog Devices AD1889"
 	select SND_AC97_CODEC
diff --git a/sound/pci/Makefile b/sound/pci/Makefile
index 5665c1e..541057b 100644
--- a/sound/pci/Makefile
+++ b/sound/pci/Makefile
@@ -77,4 +77,5 @@ obj-$(CONFIG_SND) += \
 	rme9652/ \
 	trident/ \
 	ymfpci/ \
-	vx222/
+	vx222/ \
+	sst/
diff --git a/sound/pci/sst/Makefile b/sound/pci/sst/Makefile
new file mode 100644
index 0000000..cf96b11
--- /dev/null
+++ b/sound/pci/sst/Makefile
@@ -0,0 +1,9 @@
+#
+# Makefile for Intel MID Audio drivers
+#
+EXTRA_CFLAGS=-g -DCONFIG_MRST_RAR_HANDLER
+snd-intel-sst-objs := intel_sst.o intel_sst_ipc.o intel_sst_stream.o intel_sst_interface.o intel_sst_dsp.o intel_sst_pvt.o
+snd-intelmid-objs := intelmid.o intelmid_v0_control.o intelmid_v1_control.o intelmid_v2_control.o intelmid_ctrl.o intelmid_pvt.o
+# Toplevel Module Dependency
+obj-$(CONFIG_SND_INTEL_SST) += snd-intel-sst.o
+obj-$(CONFIG_SND_INTELMID) += snd-intelmid.o
-- 
1.6.2.2

